packages/data/src/onboarding/resolvers.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * External Dependencies
+ */
+
+import { apiFetch } from '@wordpress/data-controls';
+
+/**
+ * Internal dependencies
+ */
+import { WC_ADMIN_NAMESPACE } from '../constants';
+import { setProfileItems, setError } from './actions';
+
+export function* getProfileItems() {
+	try {
+		const results = yield apiFetch( {
+			path: WC_ADMIN_NAMESPACE + '/onboarding/profile',
+			method: 'GET',
+		} );
+
+		yield setProfileItems( results, true );
```

Comment:
I don't think it's possible to return the results here since the returned results returns the existing selector state.  I asked this same question in p1589468076420800-slack-woo-start-manage and if I understood correctly, we cannot await or get the return from a resolver.

On a side note, I have an idea for a util that would let us use await with these resolvers that I may try today.  I'll open another PR if I can get things working there.

Reply:
Oh right, I thinking about how I did that with actions, not resolvers.

👍